packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.css

```diff
@@ -6,5 +6,7 @@
 
 .sort__header {
     color: #3e3e3c;
-    padding: 6px 6px 0px 0px;
+    padding-top: 6px;
```

Comment:
Should we be using the slds class for the paddings? Or is it necessary to be in px?

`slds-p-top_x-small` maybe? or thing like that?

Reply:
I tried using slds classes for padding but vertically, xx-small had the label aligned just a little too high and x-small just a little too low, so I went with px instead. Horizontally, even xx-small starts with a pretty large gap between the label and the combobox